base/intset.jl

```diff
@@ -1,339 +1,231 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-type IntSet <: AbstractSet{Int}
-    bits::Array{UInt32,1}
-    limit::Int
-    fill1s::Bool
-
-    IntSet() = new(zeros(UInt32,256>>>5), 256, false)
+immutable IntSet <: AbstractSet{Int}
+    bits::BitVector
+    IntSet() = new(fill!(BitVector(256), false))
```

Comment:
You should be able to just do `new(falses(256))`, as `falses` returns a `BitVector`.